src/BinaryRewriting/BclRewriter/BclRewriter.csproj

```diff
@@ -0,0 +1,93 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>BclRewriter</RootNamespace>
+    <AssemblyName>BclRewriter</AssemblyName>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <CLSCompliant>false</CLSCompliant>
+    <NoWarn>618</NoWarn>
```

Comment:
What obsolete warnings are you disabling? If we have some we need to use then please scope it to just the code that uses the obsolete code with `#pragma`.